modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/SettingsImpl.kt

```diff
@@ -1355,11 +1356,24 @@ class SettingsImpl @Inject constructor(
         sharedPrefs = sharedPreferences,
     )
 
-    override val useDynamicColorsForWidget: UserSetting<Boolean> = UserSetting.BoolPref(
+    override val useDynamicColorsForWidget: UserSetting<Boolean> = object : UserSetting<Boolean>(
         sharedPrefKey = "useDynamicColorsForWidget",
-        defaultValue = false,
         sharedPrefs = sharedPreferences,
-    )
+    ) {
+        override fun get(): Boolean {
+            return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                sharedPrefs.getBoolean(sharedPrefKey, false)
+            } else {
+                false
+            }
```

Comment:
I had to do this because if we synced between Android SDK 31 device that had this setting enabled the colors looked of on the device that didn't support dynamic colors.